webnn.json

```diff
@@ -345,7 +352,10 @@
       {"name": "dilations count", "type": "uint32_t", "default": 0},
       {"name": "dilations", "type": "int32_t", "annotation": "const*", "length": "dilations count", "optional": true},
       {"name": "auto pad", "type": "auto pad", "default": "explicit"},
-      {"name": "layout", "type": "input operand layout", "default": "nchw"}
+      {"name": "layout", "type": "input operand layout", "default": "nchw"},
+      {"name": "rounding type", "type": "rounding type", "default": "floor"},
+      {"name": "output sizes count", "type": "uint32_t", "default": 0},
```

Comment:
@huningxin Why does spec define the outputSizes that can be inferred with input arguments?